.github/workflows/main.yml

```
@@ -9,10 +9,23 @@ on:
 jobs:
   unit-tests:
     name: Unit tests
+    # Workaround https://github.com/nektos/act/issues/1875
     uses: ./.github/workflows/unit_tests.yml
     with:
       linux_5_9_arguments_override: "--explicit-target-dependency-import-check error -Xswiftc -strict-concurrency=complete"
       linux_5_10_arguments_override: "--explicit-target-dependency-import-check error -Xswiftc -strict-concurrency=complete"
       linux_6_0_arguments_override: "--explicit-target-dependency-import-check error"
       linux_nightly_6_0_arguments_override: "--explicit-target-dependency-import-check error"
       linux_nightly_main_arguments_override: "--explicit-target-dependency-import-check error"
+
+  benchmarks:
+    name: Benchmarks
+    # Workaround https://github.com/nektos/act/issues/1875
+    uses: ./.github/workflows/benchmarks.yml@main
```

```suggestion
    uses: ./.github/workflows/benchmarks.yml
```
Does it make sense to specify `@main` for a local workflow?

---

Oops. Missed to remove that one. I was playing around with doing it the same way that `swift-nio` does by referencing the entire repository via `swift-server/swift-kafka-client/.github/workflows/benchmarks.yml`. Looks like my search-and-replace wasn't diligent enough. Thank!